test failures in Python 3.7

The test suite passes for me with Python 3.6.5, but when run with Python 3.7.0b5 there are two failures.

```
======================================================================
FAIL: test_no_constructor_args (reader_test.TestAutoReader)
----------------------------------------------------------------------
TypeError: function missing required argument 'database' (pos 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/maxminddb-1.4.0/tests/reader_test.py", line 179, in test_no_constructor_args
    cls)
AssertionError: " 1 required positional argument|\(pos 1\) not found|takes at least 2 arguments" does not match "function missing required argument 'database' (pos 1)"

======================================================================
FAIL: test_no_constructor_args (reader_test.TestExtensionReader)
----------------------------------------------------------------------
TypeError: function missing required argument 'database' (pos 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/maxminddb-1.4.0/tests/reader_test.py", line 179, in test_no_constructor_args
    cls)
AssertionError: " 1 required positional argument|\(pos 1\) not found|takes at least 2 arguments" does not match "function missing required argument 'database' (pos 1)"

----------------------------------------------------------------------
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,6 +7,7 @@ python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7-dev
   - pypy
 
 before_install:

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -3,6 +3,11 @@
 History
 -------
 
+1.4.1
+++++++++++++++++++
+
+* Fix test failure on Python 3.7. Reported by Carl George. GitHub #35.
+
 1.4.0 (2018-05-25)
 ++++++++++++++++++
 

--- a/tests/reader_test.py
+++ b/tests/reader_test.py
@@ -175,7 +175,10 @@ class BaseTestReader(object):
         cls = self.readerClass[0]
         self.assertRaisesRegex(
             TypeError,
-            ' 1 required positional argument|\(pos 1\) not found|takes at least 2 arguments',
+            ' 1 required positional argument|'
+            '\(pos 1\) not found|'
+            'takes at least 2 arguments|'
+            'function missing required argument \'database\' \(pos 1\)',
             cls)
 
     def test_too_many_get_args(self):

